cli/api/api.go

```diff
@@ -46,7 +46,6 @@ type Options struct {
 	VarPath              string
 	ResourcePath         string
 	Zookeepers           []string
-	RemoteZookeepers     []string
```

Comment:
Should this be called JWTTTL?  I feel like it may make sense to genericize it further by calling it AuthTTL or TokenTTL